packages/eas-cli/src/project/workflow.ts

```diff
@@ -34,3 +34,16 @@ export async function resolveWorkflowAsync(
   }
   return Workflow.MANAGED;
 }
+
+export type CombinedWorkflowData = {
+  [value in Platform]: Workflow;
+};
```

Comment:
maybe inline that type in the function return value or name it sth more descriptive e.g. `WorkflowPerPlatform`, plus you probably should use `Record<Platform, Workflow>` instead